src/tpm2_quote.c

```diff
@@ -361,21 +320,15 @@ int main(int argc, char *argv[])
 
     if(argc == 1)
     {
-        showHelp(argv[0]);
+        LOG_ERR("Invalid usage, try --help for help!");
         return 0;
     }
     while((opt = getopt_long(argc,argv,optstring,long_options,NULL)) != -1)
     {
         switch(opt)
         {
-        case 'h':
-            h_flag = 1;
-            break;
-        case 'v':
-            v_flag = 1;
-            break;
         case 'k':
-            if(getSizeUint32Hex(optarg,&akHandle) != 0)
+            if(!string_bytes_get_uint32(optarg,&akHandle) != 0)
```

Comment:
now string_bytes_get_uint32() return bool directly, we should remove  " != 0" above.